k-distribution/include/kframework/builtin/domains.md

```diff
@@ -2092,15 +2097,17 @@ of the `Bytes` term).
 
 ```k
   syntax Bytes ::= Bytes "[" index: Int "<-" value: Int "]" [function, hook(BYTES.update)]
+  syntax {Width} Bytes ::= Bytes "[" index: MInt{Width} "<-" value: MInt{Width} "]" [function, hook(BYTES.updateMInt)]
```

Comment:
you did not mention the limitation of this working only for 64 and 256 bitwidths. Is it not limited to those?